src/options.html

```diff
@@ -20,6 +20,14 @@
     <input type="text" id="emailPattern" name="dateFormat" />
     <span class="help-text">E.g. jdoe+{name}@gmail.com</span>
   </div>
+  <div class="form-group">
+    <label>Name Generation</label>
+    <div class="radio">
+      <label><input type="radio" name="nameGenerator" id="nameGenerator-random" value="random"> Random names</label>
```

Comment:
This would probably be simpler as a dropdown and would avoid the gymnastics when getting which value is checked